apps/frontend-manage/src/components/sessions/creation/MultistepWizard.tsx

```diff
@@ -163,7 +163,6 @@ function MultistepWizard({
       onSubmit={handleSubmit}
       validationSchema={step.props.validationSchema}
       isInitialValid={initialValid}
-      enableReinitialize
```

Comment:
This is the prop, which caused the random bug, reinitializing the form at the wrong moment. Through a corresponding conditional statement, the cases where data is loaded for editing are handled before the initialization of the form. Therefore, there is no need for reinitialization.